events/codebuild.go

```diff
@@ -103,6 +103,8 @@ type CodeBuildEventAdditionalInformation struct {
 
 	BuildComplete bool `json:"build-complete"`
 
+	BuildNumber CodeBuildNumber `json:"build-number,omitempty"`
```

Comment:
why a new type and custom serializer? Does the builtin `int` not work for this?

```BuildNumber int `json:"build-number,omitempty"` ```

Reply:
The original value of build-number in CodeBuild JSON event is not presented as integer but float (e.g. 55.0), have to use a new type to leverage custom marshalling to convert it to `int`. Cheers.

Reply:
gotcha, makes sense to me